README.md

```diff
@@ -119,6 +119,8 @@ The precedence of configurations is as described below.
 |`--tfe-address` | `TFE_ADDRESS` | `tfe.tfe-address` | Terraform Enterprise address for states access | - |
 |`--tfe-token` | `TFE_TOKEN` | `tfe.tfe-token` | Terraform Enterprise token for states access | - |
 |`--tfe-organization` | `TFE_ORGANIZATION` | `tfe.tfe-organization` | Terraform Enterprise organization for states access | - |
+|`--gcs-bucket` | `N/A` | `gcp.gcs-buckets` | Google Cloud Storage buckets to access | - |
```

Comment:
@tristanvanthielen - are we missing the variable for the GCS bucket?

Reply:
Hi, so I did not include a variable as the gcs-bucket argument can be specified multiple times to pull states from multiple buckets. I did not really find a clean way to solve this using env variables, but if you have an idea, feel free to suggest

Reply:
`go-flags` allows to pass lists in environment variables, using the `env-delim` argument for the delimiter, for example:

```go
  GCSBuckets []string `long:"gcs-bucket" env:"GCS_BUCKET" env-delim:","`
```